SYCL/ESIMD/api/functional/type_traits.hpp

```diff
@@ -31,7 +31,7 @@ inline constexpr bool is_sycl_floating_point_v{
 
 template <typename T>
 using is_sycl_signed =
-    std::bool_constant<std::is_signed_v<T> || is_sycl_floating_point_v<T>>;
+    std::bool_constant<std::is_signed_v<T> || std::is_same_v<T, sycl::half>>;
```

Comment:
What if we would have some additional "sycl floating point" types?

Reply:
and if they will be signed types this statement should be extended

Reply:
ok, it will be reverted

Reply:
this will be reverted [here](https://github.com/intel/llvm-test-suite/pull/690/commits/be308ab37a2211afe7f091450cf5fb97c3e6346e#diff-7b267ff200ef7dbfe5e2b1fe912dcbc7a8622f4d14a995f2f683282cff5e1b49R34)